pkg/fftm/policyloop.go

```diff
@@ -317,26 +326,32 @@ func (m *manager) execPolicy(ctx context.Context, pending *pendingState, syncDel
 		}
 		pending.remove = true // for the next time round the loop
 		m.markInflightStale()
+		m.sendWSReply(mtx)
 	}
-	m.sendWSReply(mtx)
 
 	return nil
 }
 
 func (m *manager) sendWSReply(mtx *apitypes.ManagedTX) {
 	wsr := &apitypes.TransactionUpdateReply{
-		ManagedTX: *mtx,
 		Headers: apitypes.ReplyHeaders{
 			RequestID: mtx.ID,
 		},
+		Status:          mtx.Status,
+		TransactionHash: mtx.TransactionHash,
+	}
+
+	if mtx.Receipt != nil {
```

Comment:
This feels a little unnecessary to me. It extends the difference between notifications and normal just `GET` `/transactions` output, which are really different for nothing but historical reasons (EthConnect).

However, happy to leave that with you for a final decision @matthew1001

Reply:
(specifically I mean the pulling one field out of `receipt` up to the top-level, vs. the app being notified accessing it at `receipt.protocolId`)

Reply:
I think it makes sense to me, given it's expect in the notification. Suggest leaving for now and can go back round the loop in a future PR?